src/mpid/ch4/netmod/ofi/ofi_recv.h

```diff
@@ -104,12 +104,17 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_do_irecv(void *buf,
     MPIR_Request *rreq = *request;
     uint64_t match_bits, mask_bits;
     MPIR_Context_id_t context_id = comm->recvcontext_id + context_offset;
-    size_t data_sz;
+    /* Add initial value to for check in fn_fail. */
+    size_t data_sz = 0;
     int dt_contig;
     MPI_Aint dt_true_lb;
     MPIR_Datatype *dt_ptr;
     struct fi_msg_tagged msg;
-    char *recv_buf;
+    /* Set initial value for compiler uninitialization complain */
```

Comment:
This comment can be removed.  Also, if this compiler warning was occurring before this PR, it should be a separate patch.